src/main/java/svc/data/citations/filters/CitationDateFilter.java

```diff
@@ -0,0 +1,25 @@
+package svc.data.citations.filters;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.ListIterator;
+
+import svc.data.citations.datasources.CITATION_DATASOURCE;
+import svc.models.Citation;
+
+public class CitationDateFilter {
+
+	public static List<Citation> FilterDates(List<Citation> citations){
+		ListIterator<Citation> iter = citations.listIterator();
+		while(iter.hasNext()){
+			Citation citation = iter.next();
+			if (citation.citation_datasource != CITATION_DATASOURCE.MOCK){
```

Comment:
This makes me ask "why is this one data source special" (knowing the answer is that it's mock data). Letting other (non-mock) parts of your application know about the mock parts of your application is bad design.